src/libraries/Common/src/System/Net/Sockets/SocketExceptionFactory.Unix.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Net.Sockets
+{
+    internal static partial class SocketExceptionFactory
+    {
+        public static SocketException CreateSocketException(int socketError, EndPoint? endPoint)
```

Comment:
To me it looks like we don't need `endPoint` to be nullable.

Reply:
Agreed, changed into non-nullable.